content/en/docs/concepts/storage/volumes.md

```diff
@@ -1144,6 +1144,16 @@ but new volumes created by the vSphere CSI driver will not be honoring these par
 
 To turn off the `vsphereVolume` plugin from being loaded by the controller manager and the kubelet, you need to set `InTreePluginvSphereUnregister` feature flag to `true`. You must install a `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver on all worker nodes.
 
+#### Portworx CSI migration
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+The `CSIMigration` feature for Portworx is added and disabled by default in Kubernetes 1.23 and is in alpha state.
```

Comment:
How about?

```suggestion
The `CSIMigration` feature for Portworx is added but disabled by default in Kubernetes 1.23 since it's in alpha state.
```